taichi/codegen/codegen_opengl.h

```diff
@@ -30,7 +30,7 @@ class OpenglCodeGen {
 
   Program *prog_;
   Kernel *kernel_;
-  const StructCompiledResult *struct_compiled_;
+  StructCompiledResult *struct_compiled_;
```

Comment:
@archibate let's add `[[maybe_unsed]]` to `StructCompiledResult *struct_compiled_`, since it generates a warning on non-OpenGL build. Feel free to do this in any OpenGL-related PR since it's just one line deletion. Thanks!